packages/entities/entities-shared/src/components/entity-base-table/EntityBaseTable.vue

```diff
@@ -354,11 +354,14 @@ const handleSortChanged = (sortParams: TableSortParams): void => {
 
 const { setTablePreferences, getTablePreferences } = useTablePreferences()
 
-const tablePreferences = ref<UserTablePreferences>(getTablePreferences(props.preferencesStorageKey))
+// Use unique key for localStorage of user's table preferences across tables, orgs and users
+const tablePreferencesStorageKey = computed((): string => `tablePreferences-${cacheId.value}`)
```

Comment:
Using `cacheId` here as it is [derived from ](https://github.com/Kong/public-ui-components/blob/main/packages/entities/entities-shared/src/components/entity-base-table/EntityBaseTable.vue#L270-L272)`cacheIdentifier` and fallback is `preferencesStorageKey`